docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -804,6 +804,9 @@ This endpoint returns a live stream of a container's resource usage statistics.
 
       {
          "read" : "2015-01-08T22:57:31.547920715Z",
+         "pids_stats": {
```

Comment:
I think you need to remove this, it does not exist in v1.22. You should add it to v1.23.